modules/social_features/social_core/social_core.module

```diff
@@ -80,6 +81,38 @@ function social_core_modules_installed($modules) {
   }
 }
 
+/**
+ * Implements hook_entity_form_display_alter().
+ */
+function social_core_entity_form_display_alter(EntityFormDisplayInterface $form_display, array $context) {
```

Comment:
@chmez Is it a good idea to simply add "Settings" field group during the install of "social_core" module?

Reply:
Indeed. We'll need to improve it as well. Also, please, mention me through my work account.